sass/responsive/_mobile.scss

```diff
@@ -410,7 +410,17 @@
             overflow: auto;
         }
     }
-
+    .app__body {
+        .modal {
+            .settings-modal {
+                &:not(.settings-modal--tabless):not(.display--content) {
```

Comment:
Should these modals depend on '.settings-modal'? Seems like they don't need the styles provided by that class.